src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/service/PrisonOffenderEventListener.kt

```diff
@@ -29,8 +29,10 @@ class PrisonOffenderEventListener(
     when (eventType) {
       PRISONER_MERGE_EVENT_TYPE -> {
         val mergeEvent = mapper.readValue(message, HMPPSMergeDomainEvent::class.java)
-        val (removedPrisonerNumber, prisonerNumber) = mergeEvent.additionalInformation
```

Comment:
these 2 fields are defined in the opposite order in the data class so are unpacked incorrectly here!